docs/source/notes/autograd.rst

```diff
@@ -210,3 +210,108 @@ No thread safety on C++ hooks
 Autograd relies on the user to write thread safe C++ hooks. If you want the hook
 to be correctly applied in multithreading environment, you will need to write
 proper thread locking code to ensure the hooks are thread safe.
+
+Autograd for Complex Numbers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**What notion of complex derivative does PyTorch use?**
+*******************************************************
+
+PyTorch follows `JAX's <https://jax.readthedocs.io/en/latest/notebooks/autodiff_cookbook.html#Complex-numbers-and-differentiation>`_
+convention for autograd for Complex Numbers.
+
+Suppose we have a function :math:`F: ℂ → ℂ` which we can decompose into functions u and v
```

Comment:
In this section, v is:
- the function corresponding to the imaginary part of f
- the covector in `vjp` or vector in `jvp`
- the input space of f

It would be nice if we could avoid some of these overloads.

Reply:
v is the function corresponding to the imaginary part of f. not sure where did you infer the (2) and (3) from?

Reply:
2) Was from the `vjp` notation, but I guess you mean vector here. So that's fine.
3) is from your definition of VJP below: `VJP : V → (W^* → V^*)`